grid/src/io.rs

```diff
@@ -0,0 +1,104 @@
+use crate::grid::SerialGrid;
+pub use solvers_traits::grid::Geometry;
+pub use solvers_traits::grid::Grid;
+pub use solvers_traits::grid::Topology;
+use std::fs;
+
+// pub fn export_as_gmsh(grid: impl Grid, fname: String) {
+pub fn export_as_gmsh(grid: SerialGrid, fname: String) {
```

Comment:
Why does this method take ownership of the grid?

Reply:
& added